analyses/tp53_nf1_score/00-tp53-nf1-alterations.R

```diff
@@ -135,7 +149,6 @@ tp53_nf1_coding <- tp53_coding %>%
 bs_ids <- histology %>%
   filter(sample_type == "Tumor",
          experimental_strategy != "RNA-Seq") %>%
```

Comment:
```suggestion
         experimental_strategy %in% c("WGS", "WXS) %>%
```
Can we change this so the proteomics samples are not showing in output?